### PR TITLE
refactor(fastify): updated static options

### PR DESCRIPTION
**Pull request by Mojtaba-NA**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
since version 6.10.0, fastify-static added an option to specify route constraints. I've just updated the interface in platform-fastify package to reflect the latest type in fastify-static package.

[fastify-static v6.10.0 changelog](https://github.com/fastify/fastify-static/releases/tag/v6.10.0)

## Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

## Other information

### CODE REVIEW

**Review by Tony133 — APPROVED** [review]

File: packages/platform-express/multer/interceptors/any-files.interceptor.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ import { transformException } from '../multer/multer.utils';
 type MulterInstance = any;
 
 /**
- * @param localOptions 
- * 
+ * @param localOptions
+ *
```
Comment (jrobeson) [inline packages/platform-express/multer/interceptors/any-files.interceptor.ts:21]: These spacing changes are unrelated to the commit aren't they? If so, they shouldn't be in it, and rather submited as a separate PR for formatting

Comment (Mojtaba-NA) [reply packages/platform-express/multer/interceptors/any-files.interceptor.ts]: @jrobeson changed the PR to contain only the changes that I've made.
